lib/config/rules/best-practices.js

```diff
@@ -7,8 +7,6 @@ module.exports = {
   'array-callback-return': 'error',
   // Treat var statements as if they were block scoped
   'block-scoped-var': 'warn',
-  // Enforce that class methods utilize this
-  'class-methods-use-this': 'warn',
```

Comment:
Can we please keep the rule but just set it to `off`?

Reply:
Sure thing